src/lib/core/backtrace.cc

```diff
@@ -358,7 +366,7 @@ __asm__ volatile(
 	"\tldp d14, d15, [x0, #16 * 9]\n"
 	"\tadd sp, x0, #8 * 20\n"
 	:
-	: "r" (unw_context), "r" (coro_ctx), "i" (unw_getcontext_f)
+	: "r" (unw_context), "r" (coro_ctx)
```

Comment:
I suggest to use `"S"` constraint here.
From the [gcc guide](https://gcc.gnu.org/onlinedocs/gcc/Machine-Constraints.html) for AArch64 family:
> S

    An absolute symbolic address or a label reference

I suspect that `coro_unwcontext()` implementation for the regular ARM should be changed this way too.
```suggestion
	: "r" (unw_context), "r" (coro_ctx), "S" (unw_getcontext_f)
```

Reply:
Accepted.